typing/jkind_axis.mli

```diff
@@ -12,29 +12,8 @@
 (*                                                                        *)
 (**************************************************************************)
 
-(** The common interface for jkind axes *)
-module type Axis_s = sig
-  type t
-
-  val max : t
-
-  val min : t
-
-  val equal : t -> t -> bool
-
-  val less_or_equal : t -> t -> Misc.Le_result.t
-
-  val le : t -> t -> bool
-
-  val meet : t -> t -> t
-
-  val join : t -> t -> t
-
-  val print : Format.formatter -> t -> unit
-end
-
-(** Adapt a [Lattice] to be an [Axis_s] *)
-module Of_lattice (L : Mode_intf.Lattice) : Axis_s with type t = L.t
+(** Re-export *)
+module type Lattice = Mode_intf.Lattice
```

Comment:
The extra functions `equal` and `less_or_equal` are not used by `mode.ml` or `solver.ml`; they also require some extra invariants between `le` and `equal` and `less_or_equal` that are not guaranteed. Maybe we can instead have:
```
Misc.Le_result.less_or_equal : ('a -> 'a -> bool) -> 'a -> 'a -> Misc.Le_result.t
Misc.Le_result.equal : ('a -> 'a -> bool) -> 'a -> 'a -> bool
```
And then instead of `Mode.Uniqueness.less_or_equal a b` we write `Misc.Le_result.less_or_equal Mode.Uniqueness.le a b`.

Reply:
Ok